torch/testing/_internal/common_nn.py

```diff
@@ -3689,7 +3689,7 @@ def fractional_max_pool3d_test(test_case):
     ),
     dict(
         module_name='TransformerEncoderLayer',
-        constructor_args=(4, 2, 8, 0.0, 'gelu'),
+        constructor_args=(4, 2, 8, 0.0, F.gelu),
```

Comment:
Are there any tests left that test legacy string activation?

Reply:
There's the test in `test_nn.py` that iterates over string, functional, and module forms- is that sufficient or should I leave one of the others with string activation?